server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/LocalDelivery.java

```diff
@@ -77,6 +77,8 @@ public void init() throws MessagingException {
                 .metric(metricFactory.generate(LOCAL_DELIVERED_MAILS_METRIC_NAME))
                 .build())
             .consume(getInitParameter("consume", true))
+            .ignoreError(getInitParameter("ignoreError", false))
+            .errorProcessor(getInitParameter("errorProcessor", Mail.ERROR))
```

Comment:
Please use the standard `onMailetException` property to do this.

CF https://james.staged.apache.org/james-distributed-app/3.7.0/configure/mailetcontainer.html#_error_handling